org.videolan.VLC.Plugin.makemkv.json

```diff
@@ -0,0 +1,90 @@
+{
+  "id": "org.videolan.VLC.Plugin.makemkv",
+  "build-extension": true,
+  "runtime": "org.videolan.VLC",
+  "runtime-version": "stable",
+  "branch": "3-1.6",
+  "sdk": "org.freedesktop.Sdk//1.6",
```

Comment:
1.6? you should use 18.08 instead here

Reply:
I don't think so, org.videolan.VLC is still on 1.6, via Qt 5.11 Sdk.